src/index.ts

```diff
@@ -149,14 +145,13 @@ const scrubRoute = (x: Route) => {
 
 const scrubRoutes = (x: Route[]) => x.map(scrubRoute)
 
-function parse_legacy_tx_format(tx: AgnosticStdTx): ProtoTx {
-  const msgOrMsgs = tx.msg ?? tx.msgs
-  if(!msgOrMsgs) throw new Error('msgs array improperly formatted!')
+function parse_legacy_tx_format(tx:  StdTxWithMsgs): ProtoTx {
+  if(!tx.msgs) throw new Error('msgs array improperly formatted!')
```

Comment:
`msgs` now required so could rely on typescript and get rid of this check

Reply:
👍